lib/simplifyExpression/basicsSearch/removeDivisionByOne.js

```diff
@@ -1,38 +1,38 @@
-const clone = require('../../util/clone');
+const {rewriteNode} = require('math-rules');
 
 const ChangeTypes = require('../../ChangeTypes');
-const Negative = require('../../Negative');
+const defineRuleString = require('../../util/defineRuleString');
 const Node = require('../../node');
 
+const simplifyDoubleUnaryMinus = require('./simplifyDoubleUnaryMinus');
+const {build, query} = require('math-nodes');
+
+// e.g. 2/-1 -> -2
+const DIVISION_BY_NEGATIVE_ONE = defineRuleString('#a / -1', '-#a');
+
+// e.g. 2/1 -> 2
+const DIVISION_BY_ONE = defineRuleString('#a / 1', '#a');
+
 // If `node` is a division operation of something by 1 or -1, we can remove the
 // denominator. Returns a Node.Status object.
 function removeDivisionByOne(node) {
-  if (node.op !== '/') {
-    return Node.Status.noChange(node);
-  }
-  const denominator = node.args[1];
-  if (!Node.Type.isConstant(denominator)) {
-    return Node.Status.noChange(node);
+  let newNode = null;
+  newNode = rewriteNode(DIVISION_BY_ONE, node);
+  if (newNode) {
+    return Node.Status.nodeChanged(
+      ChangeTypes.DIVISION_BY_ONE, node, newNode);
   }
-  let numerator = clone(node.args[0]);
-
-  // if denominator is -1, we make the numerator negative
-  if (parseFloat(denominator.value) === -1) {
-    // If the numerator was an operation, wrap it in parens before adding -
-    // to the front.
-    // e.g. 2+3 / -1 ---> -(2+3)
-    if (Node.Type.isOperator(numerator)) {
-      numerator = Node.Creator.parenthesis(numerator);
-    }
-    const changeType = Negative.isNegative(numerator) ?
-      ChangeTypes.RESOLVE_DOUBLE_MINUS :
-      ChangeTypes.DIVISION_BY_NEGATIVE_ONE;
-    numerator = Negative.negate(numerator);
-    return Node.Status.nodeChanged(changeType, node, numerator);
+
+  const numerator = node.args[0];
+  if (query.isNeg(numerator)) {
+    return simplifyDoubleUnaryMinus(build.neg(numerator));
```

Comment:
And the `math-steps` codebase combines `REMOVE_DIVISION_BY_ONE` and `REMOVE_DIVISION_BY_NEGATIVE_ONE` into one file.